checkmarx-ltd/VisualStudio
# Plug 29

**Changes**
VS plugin added vulnerability description and a link to Codebashing.

**Short Description Tests**

**_Test Case 1_**

1. Login with VS plugin
2. Bind project
3. Open CxViewer tree -> click on any severity 
4. Open CxViewer Result and verify short description

**_Test Case 2_**

1. Login with VS plugin
2. Bind project
3. Open CxViewer tree -> click on any severity
4. Click on any results in CxViewer Result and verify short description

**_Codebashing Test_**

**_Test Case 1_**

1. Login with VS plugin
2. Bind project
3. Open CxViewer tree -> click on any severity
4. Open CxViewer Result and check Codebashing link, Codebashing link tooltip and onclick it redirects to codebashing link
